src/jit/Backend.cpp

```diff
@@ -392,6 +392,18 @@ static void emitBrTable(sljit_compiler* compiler, BrTableInstruction* instr)
     context->branchTableOffset = reinterpret_cast<sljit_uw>(target);
 }
 
+static void emitMove32(sljit_compiler* compiler, Instruction* instr)
+{
+    Operand* operands = instr->operands();
+    JITArg args[2];
+
+    for (unsigned i = 0; i < 2; ++i) {
```

Comment:
Since this loop only has two steps, I would put operandToArg twice. Similar to the 64 bit case.